documentation

crdanielbusch opened the pull request:
Work in progress

## Description

## Checklist

Please confirm that this pull request has done the following:

- [ ] Tests added
- [ ] Documentation added (where applicable)
<!--- Below to be added back once we have a changelog --> 
<!--- - [ ] Changelog item added to `changelog/`) -->

## Notes

crdanielbusch commented on README.md:
````diff
@@ -132,39 +146,75 @@ To calculate emissions for all layers, run `omPrior.py` with a start and end dat
+
+```
+docker run --rm -v </your/path/to/openmethane-prior>:/opt/project openmethane-prior python scripts/omDownloadInputs.py
+```
````
Is this actually relevant for users? Or do you only need docker if you run the application on a server like we do?

lewisjared replied:
True. Good point. If we make developer docs then docker usage can go there, but for now assume that you are running locally

crdanielbusch commented on README.md:
````diff
@@ -36,70 +36,90 @@ The Open Methane prior can be installed from source into a virtual environment w
+To download all the required input files, run:
+
+```console
+make download
````
I replaced the typed commands with the make commands and added a note that the full commands can be found in Makefile. We can also do it the other way round

lewisjared commented:
Nice stuff. Can we add in some docs about the input/output directories

crdanielbusch commented:
> Nice stuff. Can we add in some docs about the input/output directories

Added a basic description of the data directories.
And changed the terminology in the .env file to make it easier to find in the README.

lewisjared reviewed (approved):
Opps I didn't press the approve button again
